src/elements/content-sidebar/SidebarClassification.scss

```diff
@@ -0,0 +1,6 @@
+@import '../common/variables';
+
+.bdl-SidebarClassification-editBtn {
```

Comment:
Should this be prefixed with `.bcs`, instead? It doesn't look to be meant as a common component.

Reply:
Actually on second thoughts, everything within this project should be bdl as we have been doing at other places. Having more than 1 namespace just adds more than it achieves. Right now there is bdl, be, bcs etc. A single namespace should suffice. And if be/bcs will carry styling, it should be bdl-be/bcs if really needed or one can just switch to mixins instead.